packages/veritone-react-common/src/components/Parts/Buttons/PillButton/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import defaultStyles from './styles.scss';
+
+export default class PillButton extends Component {
+	static propTypes = {
+		label: PropTypes.string,
+		info: PropTypes.string,
+		style: PropTypes.instanceOf(Object),
+		gapStyle: PropTypes.instanceOf(Object),
+		labelStyle: PropTypes.instanceOf(Object),
+		infoStyle: PropTypes.instanceOf(Object),
+		className: PropTypes.string,
+		gapClassName: PropTypes.string,
+		labelClassName: PropTypes.string,
+		infoClassName: PropTypes.string,
+		highlight: PropTypes.bool,
+		onClick: PropTypes.func
+	};
+
+	static defaultProps = {
```

Comment:
doesn't look like these defaults are needed -- it's no problem to pass `undefined` in the case that they weren't provided